x/ccv/consumer/keeper/keeper.go

```diff
@@ -384,7 +429,11 @@ func (k Keeper) DeleteCCValidator(ctx sdk.Context, addr []byte) {
 	store.Delete(types.CrossChainValidatorKey(addr))
 }
 
-// GetAllCCValidator returns all cross-chain validators
+// GetAllCCValidator returns all cross-chain validators.
+//
+// Note that the cross-chain validators are stored under keys with the following format:
+// CrossChainValidatorBytePrefix | address
+// Thus, the iteration is in ascending order of addresses.
```

Comment:
Why is order of iteration important if you return a slice and not an iterator? I'm not sure the implications from iteration should be applied to a slice - it can be sorted in any appropriate way by the caller.

Are we adding a comment about the presumed order of iteration on all functions from here onward?

Reply:
It's important as some code could assume a certain order of the elements in the returned slice. None of the functions return iterators. They iterate over a set of keys and apply a callback function to each entry. The callback here is implicit, i.e., collect all entries in a slice. 

> Are we adding a comment about the presumed order of iteration on all functions from here onward?

No. Just on those ones that have an iterator inside.